### DISCUSSION TITLE
Welcome to sl3 Discussions!

### DISCUSSION THREAD

**Discussion by jeremyrcoyle** [tlverse/sl3#361] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frbl** [comment]
Hey Everyone, my name is Frank, and I've spent a few months together with you at UC Berkeley. I'm just writing the message as I wanted to let you all know that I'm still closely following the progress on this project, and I think its extremely cool how much functionality you've added over the past months. Keep up the good work!

All the best,
Frank Blaauw

**Reply by jeremyrcoyle** [reply]
Thanks for your support and encouragement!
